docs/messaging/check-submission/readme.md

```diff
@@ -0,0 +1,42 @@
+# Check Submission Payload
+
+The submitted check payload is [compressed to UTF-16](https://github.com/pieroxy/lz-string) before sending.  Once received it is hydrated for validation and marking.
+
+An example payload schema can be found [here](./check-payload.json)
+
+## Pupil Events
+
+| Event Name | Description |
+| --------- | ---------------- |
+| AppError | Application error in the pupil's browser |
+| AppHidden | "This event is triggered when the pupil-app changes from being in the foreground to the background, e.g. when switching to another tab/window in the browser, or to another application." |
```

Comment:
This file has inconsistent use of quotes around the description.  Best to remove them.